vortex-filtering/include/vortex_filtering/filters/README.md

```diff
@@ -1,38 +1,228 @@
 # Filters
-This folder contains various filters
+This folder contains various filters for state estimation. Th
```

Comment:
Th?

Reply:
The filters are based on the Kalman filter and its extensions, and are designed to work with the models in the `vortex::models` namespace.